app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -64,9 +64,9 @@ def dest_folder
 
     host_dc = dest_host.parent_datacenter || dest_host.ems_cluster.parent_datacenter
 
-    # Pick 'Discovered virtual machine' or its parent folder in the destination datacenter
+    # Pick the parent folder in the destination datacenter
     vm_folder = "#{host_dc.folder_path}/vm"
-    find_folder("#{vm_folder}/Discovered virtual machine", host_dc) || find_folder(vm_folder, host_dc)
+    find_folder(vm_folder, host_dc)
```

Comment:
Minor, but you do not need the `vm_folder` variable anymore since it is only used once.

`find_folder("#{host_dc.folder_path}/vm", host_dc)`